wfl/fit/ace.py

```diff
@@ -14,16 +14,18 @@
 import ase.io
 from ase.stress import voigt_6_to_full_3x3_stress
 
-from wfl.configset import ConfigSet
+from wfl.configset import OutputSpec 
 from wfl.autoparallelize.utils import get_remote_info
 from wfl.fit.utils import ace_fit_jl_path
+from wfl.utils.configs import find_isolated_atoms
 
 from expyre import ExPyRe
 import wfl.scripts
 
 def fit(fitting_configs, ACE_name, ace_fit_params, ace_fit_command=None,
         ref_property_prefix='REF_', skip_if_present=False, run_dir='.', dry_run=False,
-        verbose=True, remote_info=None, remote_label=None, wait_for_results=True):
+        verbose=True, remote_info=None, remote_label=None, wait_for_results=True,
+        isolated_atom_info_key="config_type", isolated_atom_info_value="default"):
```

Comment:
In some ways this can be made cleaner by allowing `isolated_atom_info_value` to be a list/tuple, and doing `if atoms.info[isolated_atom_info_key] in isolated_atom_info_value:` instead of checking for `==`, (with the default being `["isolated_atom", "IsolatedAtom"]`.  We'd have to watch for people passing a string and turn it into a single-item list of strings.

Do you think that's nicer?

Reply:
I just saw that you _are_ doing `in`. I think that without converting a plain string to a list of strings that won't work quite right: `"i" in "isolated_atom"` evaluates to `True`, but I don't think we want `config_type="i"` to match.

Reply:
What about line 28 of [wfl/utils/configs.py](https://github.com/libAtoms/workflow/pull/174/files#diff-6a6b28296323e77fb34d605cef7aa490cc5dad83221289749474f111811c2e19)? How/is what you suggesting different?

Reply:
No, you're right, you do it properly in line 30.